rollup.config.js

```diff
@@ -65,22 +72,38 @@ const deps = Object.keys(pkg.dependencies || {});
 const peerDeps = Object.keys(pkg.peerDependencies || {});
 const defaultExternal = deps.concat(peerDeps).concat(ignoredExternals);
 
-// We need to define 2 separate configs (`esm` and `cjs`) so that each can be
-// further customized.
-const config = {
-  input: 'src/index.js',
-  external: defaultExternal,
-  output: [
-    {
-      file: pkg.main,
-      format: 'cjs',
-    },
-    {
-      file: pkg.module,
-      format: 'esm',
+const createConfig = cliArgs => {
+  return {
+    input: cliArgs.input,
+    external: defaultExternal,
+    treeshake: { moduleSideEffects: false },
+    onwarn(warning, warn) {
+      // skip certain warnings
+      if (
+        warning.code === 'UNUSED_EXTERNAL_IMPORT' &&
+        (warning.source === 'react' || warning.source === '@emotion/core')
+      )
+        return;
+
+      // always throw when unresolved import
+      if (warning.code === 'UNRESOLVED_IMPORT') {
```

Comment:
because Lerna hides warnings, it's a good idea to fail early if a package is missing a dep.